src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -1458,31 +1438,23 @@ internal static bool IsSingleFileSystemAbsolutePath(string path)
         /// Determines if the given path is relative or absolute.
         /// </summary>
         /// <param name="path">
-        /// The path used in the determination
+        /// The path used in the determination.
         /// </param>
         /// <returns>
-        /// true if the path is an absolute path, false otherwise.
+        /// True if the path is an absolute path, false otherwise.
         /// </returns>
         /// <exception cref="ArgumentNullException">
         /// If <paramref name="path"/> is null.
         /// </exception>
-        internal static bool IsAbsolutePath(string path)
+        internal static bool IsAbsolutePath(ReadOnlySpan<char> path)
         {
-            // Verify parameters
-
-            if (path == null)
-            {
-                throw PSTraceSource.NewArgumentNullException(nameof(path));
-            }
-
```

Comment:
Should we update/remove the exception doc comment?

Reply:
Done.